oe_paragraphs.post_update.php

```diff
@@ -125,3 +126,57 @@ function oe_paragraphs_post_update_10004(array &$sandbox): void {
     $display->save();
   }
 }
+
+/**
+ * Installs Banner paragraph.
+ */
+function oe_paragraphs_post_update_10005(&$sandbox): void {
+  // Install required maxlength module.
+  \Drupal::service('module_installer')->install(['maxlength']);
+
+  // Add paragraph form mode variants.
+  $form_modes = [
+    'paragraph.oe_banner_image' => 'Image banner',
+    'paragraph.oe_banner_image_shade' => 'Image shade banner',
+    'paragraph.oe_banner_primary' => 'Primary banner',
+  ];
+  foreach ($form_modes as $id => $label) {
+    $form_mode = EntityFormMode::create([
+      'id' => $id,
+      'label' => $label,
+      'targetEntityType' => 'paragraph',
+    ]);
+    $form_mode->save();
+  }
+
+  // Add Banner paragraph.
+  $storage = new FileStorage(drupal_get_path('module', 'oe_paragraphs') . '/config/post_updates/10005');
+
+  \Drupal::entityTypeManager()->getStorage('paragraphs_type')
+    ->create($storage->read('paragraphs.paragraphs_type.oe_banner'))
+    ->save();
+
+  $field_config = [
+    'field.storage.paragraph.field_oe_banner_type',
+    'field.field.paragraph.oe_banner.field_oe_title',
+    'field.field.paragraph.oe_banner.field_oe_text',
+    'field.field.paragraph.oe_banner.field_oe_link',
+    'field.field.paragraph.oe_banner.field_oe_image',
+    'field.field.paragraph.oe_banner.field_oe_banner_type',
+    'core.entity_form_display.paragraph.oe_banner.oe_banner_primary',
+    'core.entity_form_display.paragraph.oe_banner.oe_banner_image_shade',
+    'core.entity_form_display.paragraph.oe_banner.oe_banner_image',
+    'core.entity_form_display.paragraph.oe_banner.default',
+    'core.entity_view_display.paragraph.oe_banner.default',
+  ];
+
+  $config_manager = \Drupal::service('config.manager');
+  $entity_manager = \Drupal::entityTypeManager();
+  foreach ($field_config as $config) {
+    $config_record = $storage->read($config);
+    $entity_type = $config_manager->getEntityTypeIdByName($config);
```

Comment:
Similar to my comment above, I think it would be helpful if we named this $config_entity_type so that is easily understood we are dealing with configuration entities and not content entities (but this is very nitpicky so feel free to ignore it.)